#171602012 - Ender Yılmaz
#Bilişim Sistemleri Mühendisliği
#Elektronik Komponent Kayıt Kontrol Sistemi
from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_menu(object):
    def setupUi(self, menu):
        menu.setObjectName("menu")
        menu.resize(775, 214)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("buton_resim/anakart.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        menu.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(menu)
        self.centralwidget.setObjectName("centralwidget")
        self.direnc_btn = QtWidgets.QPushButton(self.centralwidget)
        self.direnc_btn.setGeometry(QtCore.QRect(10, 50, 181, 81))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setBold(True)
        font.setWeight(75)
        self.direnc_btn.setFont(font)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("buton_resim/direnc.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.direnc_btn.setIcon(icon1)
        self.direnc_btn.setIconSize(QtCore.QSize(64, 64))
        self.direnc_btn.setObjectName("direnc_btn")
        self.bobin_btn = QtWidgets.QPushButton(self.centralwidget)
        self.bobin_btn.setGeometry(QtCore.QRect(390, 50, 181, 81))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setBold(True)
        font.setWeight(75)
        self.bobin_btn.setFont(font)
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("buton_resim/bobin.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.bobin_btn.setIcon(icon2)
        self.bobin_btn.setIconSize(QtCore.QSize(64, 64))
        self.bobin_btn.setObjectName("bobin_btn")
        self.kondansator_btn = QtWidgets.QPushButton(self.centralwidget)
        self.kondansator_btn.setGeometry(QtCore.QRect(200, 50, 181, 81))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setBold(True)
        font.setWeight(75)
        self.kondansator_btn.setFont(font)
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap("buton_resim/kondansator.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.kondansator_btn.setIcon(icon3)
        self.kondansator_btn.setIconSize(QtCore.QSize(64, 64))
        self.kondansator_btn.setObjectName("kondansator_btn")
        self.komponent_btn = QtWidgets.QPushButton(self.centralwidget)
        self.komponent_btn.setGeometry(QtCore.QRect(580, 50, 181, 81))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setBold(True)
        font.setWeight(75)
        self.komponent_btn.setFont(font)
        self.komponent_btn.setIcon(icon)
        self.komponent_btn.setIconSize(QtCore.QSize(64, 64))
        self.komponent_btn.setObjectName("komponent_btn")
        self.cikis_btn = QtWidgets.QPushButton(self.centralwidget)
        self.cikis_btn.setGeometry(QtCore.QRect(680, 140, 81, 41))
        font = QtGui.QFont()
        font.setFamily("Verdana")
        font.setBold(True)
        font.setWeight(75)
        self.cikis_btn.setFont(font)
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap("buton_resim/cikis.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.cikis_btn.setIcon(icon4)
        self.cikis_btn.setIconSize(QtCore.QSize(32, 32))
        self.cikis_btn.setObjectName("cikis_btn")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(120, 10, 531, 41))
        font = QtGui.QFont()
        font.setFamily("Furore")
        font.setPointSize(14)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(570, 160, 101, 21))
        font = QtGui.QFont()
        font.setFamily("Furore")
        font.setPointSize(10)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(580, 140, 81, 21))
        font = QtGui.QFont()
        font.setFamily("Furore")
        font.setPointSize(11)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        menu.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(menu)
        self.statusbar.setObjectName("statusbar")
        menu.setStatusBar(self.statusbar)

        self.retranslateUi(menu)
        QtCore.QMetaObject.connectSlotsByName(menu)

    def retranslateUi(self, menu):
        _translate = QtCore.QCoreApplication.translate
        menu.setWindowTitle(_translate("menu", "Elektronik Komponent Kayıt Kontrol Sistemi"))
        self.direnc_btn.setText(_translate("menu", "Direnç\n"
"Hesapla"))
        self.bobin_btn.setText(_translate("menu", "Bobin\n"
"Hesapla"))
        self.kondansator_btn.setText(_translate("menu", "Kondansatör\n"
"Hesapla"))
        self.komponent_btn.setText(_translate("menu", "Komponent\n"
"Kayıt-Kontrol"))
        self.cikis_btn.setText(_translate("menu", "Çıkış"))
        self.label.setText(_translate("menu", "Elektronik Komponent Kayit Kontrol Sistemi"))
        self.label_2.setText(_translate("menu", "Ender Yilmaz"))
        self.label_3.setText(_translate("menu", "171602012"))